 metatag_views/metatag_views.module | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/metatag_views/metatag_views.module b/metatag_views/metatag_views.module
index f3afe72..2cc5258 100644
--- a/metatag_views/metatag_views.module
+++ b/metatag_views/metatag_views.module
@@ -102,6 +102,11 @@ function metatag_views_page_alter(&$page) {
     // Load the meta tags for this view.
     $metatags = $view->display_handler->get_option('metatags');
 
+    // Attempt to initialize values based on the default configuration.
+    if (empty($metatags)) {
+      $metatags = metatag_config_load_with_defaults('view');
+    }
+
     // Only proceed if there's something to work with.
     if (!empty($metatags) && is_array($metatags)) {
       // If meta tags were found but they're not nested for the language, fix
